qa-visualiser/src/fetch_sections.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+
+import requests
+from dotenv import load_dotenv
+from pydantic import ValidationError
+from requests.exceptions import JSONDecodeError, RequestException
+
+from src.models import Section
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+
+
+def fetch_sections() -> list[Section]:
```

Comment:
Python noob question: I though this had to be `List[Type]`, where `List` is from typing. Does this still work with type checking? If so what's the point of the former?

Reply:
Nah, in python 3.9 they introduced annotations for collections like `list` and `dict` so you no longer need to import `List` from typing - importing them from typing is now deprecated and being removed next year

Reply:
They introduced a lot of good stuff in 3.10 as well so now you do `TypeA | TypeB` rather than `Union[TypeA, TypeB]` and `TypeA | None` instead of `Optional[TypeA]`